### ISSUE TITLE
When using the Flowbite header and making it sticky, inputs in body overlaps header due to z-index

### ISSUE THREAD

**Issue by cududa** [themesberg/flowbite-react#529]
**Describe the bug**
When I use the default Flowbite header and make it sticky at the top, then add the Flowbite default Card with Inputs (the login card example), on scroll, the input field overlaps the header.

**To Reproduce**
1. Create a Nav.js component using the **Navbar with dropdown** from here https://flowbite-react.com/navbars
2. Add `className= "sticky top-0 w-full justify-between px-4"` to the `<Navbar> `config
3. Create a Body template like so, calling your Flowbite header:
```
function Body(props) {
	return (
		<div>
			<Nav />
			<div>
				{props.children}
			</div>
			<Footer />
		</div>
	);
}
```
5. Create a page like Login.js using **Card with form inputs** from https://flowbite-react.com/card
6. Scroll the login page up and the text input will overlay the header.
7.  Adding z-50 to the nav classname fixes

**Expected behavior**
It's a bit unnerving to possibly pay $800 for a professional library when the provided (free) card component and navigation bar aren't tested against one another, and the documentation doesn't mention the need to manage z-index

**Screenshots**
![image](https://user-images.githubusercontent.com/6423489/211925932-7c6f35ae-58b3-438d-9008-e6890f565848.png)

**Desktop (please complete the following information):**
 - OS: macOS latest
 - Browser: Chrome
 - Version: 108.0.5359.124

**Smartphone (please complete the following information):**
 - Device: Untested
 - OS: Untested
 - Browser Untested
 - Version Untested

**Additional context**

**Comment by cududa** [comment]
> @cududa it is important to say that flowbite-react isn't paid at all. It is MIT licensed, free and developed and maintained by the community.
> 
> Basically, I think that the issue that you are having is 'cause when you use `className` you are overwriting the theme style you need to make it important, so... prefix you classes with `!` or use the theme to customize it.

There is a paid version with more components that we're looking at purchasing. My point is mainly that it's a bit unnerving that the components might not be all tested to work together, and we'll have to debug as we build.

**Comment by zoltanszogyenyi** [comment]
Hey @cududa @rluders,

@rluders is right in saying that the Flowbite React library is free and open-source under the MIT license. The pro version grants you access to more components including the [dashboard interface](https://flowbite.com/react-admin-dashboard-pro/preview/) which we recently developed hiring one of our community members. Generally speaking all of the libraries in the ecosystem of Flowbite are open-source, including the main one with vanilla JS/TS, except for the pro version resources which are detailed in the [pricing page](https://flowbite.com/pro/#pricing).

That being said, the Flowbite React library is still in development and based on my knowledge (@rluders correct me if I'm wrong) we're pretty close to launching a new theming option that can be passed to components individually - and this will also be probably documented.

I think this is the type of issues that have to be tackled using that new option (discussed here https://github.com/themesberg/flowbite-react/issues/443) to prevent the need to use `!import` to overwrite classnames.

**Comment by rluders** [comment]
@zoltanszogyenyi correct, the theme per component should be released at 0.4.0 version. Only few components are missing the theme overwriting option, and we are close to release.

@cududa I really appreciate the issue reported, the project is free and open source, feel free to contribute with a PR fixing it. We are trying the best to make the library as best as possible and we count with the help of the community to achieve it. As I said, I do appreciate the issue report but the tone that you used was far from adequate.

**Comment by rluders** [comment]
Thank you for reporting this issue. However, we regret to inform you that we won't be able to address it for the following reasons:

- The reported issue refers to an unsupported old version of our project.
- Our team was unable to reproduce the problem with the information provided.
- Insufficient information was provided to conduct a thorough investigation.

We recommend upgrading to the latest version and submitting a new issue if the problem persists. Thank you for your understanding.
